tests/unit/restapi/v1/test_artifact.py

```diff
@@ -252,6 +290,87 @@ def test_artifacts_get_all(
     assert_retrieving_artifacts_works(client, expected=artifacts_expected_list)
 
 
+@pytest.mark.parametrize(
+    "sortBy, descending , expected",
+    [
+        (
+            None,
+            None,
+            [
+                "Model artifact.",
```

Comment:
Pushed up the fix for tags, I had to reference `models.Tag.last_modified_on` instead of `models.Resource.last_modified_on` like all the others.

As for the date columns, I just arbitrarily added them to the tables that were less crowded.  We can definitely talk about finalizing the columns or adding some functionality for allowing the user to add/hide columns in a future task.

Reply:
> Can we used a list of IDs for expected instead of choosing an arbitrary field? This comment applies to the sort tests as well.

Sure, just to clarify, we should still sort by `name` and `created_on`, but have the expected list be the ID's?  So something like this?

```
@pytest.mark.parametrize(
    "sortBy, descending , expected",
    [
        (None, None, ["1", "2", "3"]),
        ("name", True, ["3", "2", "1"]),
        ("name", False, ["1", "2", "3"]),
        ("createdOn", True, ["3", "2", "1"]),
        ("createdOn", False, ["1", "2", "3"]),
    ],
)
```

Reply:
Yup, except they should be ints. You can compare them against resource["id"]. Using the IDs is better since they are unique. It also makes all of these parameterized sorting tests more consistent and less verbose.

Reply:
```
@pytest.mark.parametrize(
    "sortBy, descending , expected",
    [
        (None, None, [7, 8, 9]),
        ("name", True, [8, 7, 9]),
        ("name", False, [9, 7, 8]),
        ("createdOn", True, [9, 8, 7]),
        ("createdOn", False, [7, 8, 9]),
    ],
)
```

I updated and pushed just the queues test above which passes.  Is printing out the registered items the only way to find the id's?  Also the only drawback I see to using id's is that the id value is unrelated to sorting and if that logic were to change, this would fail.  For example, I don't know why the id's start at 7.  But let me know if this is fine and I'll copy it to the rest of the tests.

Reply:
@keithmanville I updated all the sort tests to use IDs as discussed on the call.  Let me know if any other issues.